prompt_toolkit/key_binding/bindings/emacs.py

```diff
@@ -433,6 +414,8 @@ def unshift_move(event: E) -> None:
             Keys.ControlShiftRight: "forward-word",
             Keys.ControlShiftHome: "beginning-of-buffer",
             Keys.ControlShiftEnd: "end-of-buffer",
+            Keys.EscapeShiftLeft: "backward-word",
```

Comment:
So, here we should turn it into a ``Dict[Tuple[Union[Keys, str], ...], str]`` (keys become tuples). and extract that from the key_sequence.